src/shapirowilk.jl

```diff
@@ -0,0 +1,248 @@
+export ShapiroWilkTest
+
+#=
+From:
+PATRICK ROYSTON
+Approximating the Shapiro-Wilk W-test for non-normality
+*Statistics and Computing* (1992) **2**, 117-119
+DOI: [10.1007/BF01891203](https://doi.org/10.1007/BF01891203)
+=#
+
+# TODO: Rerun simulation and polynomial fitting
```

Comment:
Is this still relevant? Better not leave TODOs in the code.

Reply:
It is still relevant, but that's something for another time; I removed the TODO